stable/redis-ha/templates/_configs.tpl

```diff
@@ -231,7 +231,15 @@
     {{- $fullName := include "redis-ha.fullname" . }}
     {{- $replicas := int .Values.replicas }}
     {{- range $i := until $replicas }}
+    for loop in $(seq 0 60); do
```

Comment:
60 seconds seems a tad too long here. Can you check the actual time it takes for your system to resolve the services? 

We'd also want to document this somewhere because a lot of people may ask why this takes so long when they have similar slowness in scheduling services etc.

Reply:
Sure, note that this loop is not mandatory, since i added an exit 1 if the announce ip is not found : that is forcing failing init container and k8s will reschedule it later.

This loop is only there to avoid the k8s reschedule and make a faster init in case of slowness service creation.

For our env, just 2-3 sec are OK, i can set it to 10 sec.
I'll add a comment in the documentation if you want too

Reply:
Just to be clear, the conditions and everything surrounding it are great, it's just those experiencing a similar delay in scheduling, I don't think it's good to have this looping 60 seconds and them sitting there wondering why things aren't working, so a 10-20 seconds period would probably be more acceptable.

A little section in the README would be perfect, thank you!

Reply:
added some doc in the README (dont know is this is the right location).
Updated to 10 sec max, since it seems the best value for our case